tests/support/cobs/issue.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { RadiclePeer } from "@tests/support/peerManager.js";
+
+export async function createIssue(
```

Comment:
We could also call the methods `create`, `changeState` and then import the module with:
```
import * as issue from "./issue.ts"

issue.create
issue.changeState
```